src/client/components/utils/index.js

```diff
@@ -0,0 +1,6 @@
+import { h } from "preact";
+
+const ConditionWrap = ({ condition, children, otherwise }) =>
```

Comment:
👍

Reply:
I agree, but `else` is reserved word, I will fix that but I think something like
```
<If condition={Boolean} then=smthing otherwise=smthing />
```